### ISSUE TITLE
Content filtering

### ISSUE THREAD

**Issue by Avol-V** [aigoncharov/telegram-to-rss#7]
What about adding the ability to filter content? Quite often, there may be advertising messages in the channels that could be filtered out and not added to the RSS feed.
That is, to make a set of rules with regular expressions for checking content and in generate_feed to check their match with feed_entry.message. If it matches, then skip it.

**Comment by fxlrnrpt** [comment]
Wouldn't it be easier to do on the client side? Most of the RSS readers support powerful filtering out-of-the-box

**Comment by Avol-V** [comment]
> Wouldn't it be easier to do on the client side? Most of the RSS readers support powerful filtering out-of-the-box

For example, in [miniflux](https://miniflux.app/) I have only global content-based filters, but sometimes you need a personal filter for a channel.

**Comment by fxlrnrpt** [comment]
Fair enough. I think we could add a file with the filter config. Happy to accept a PR for it!
